runtime/doc/api.txt

```diff
@@ -773,8 +773,8 @@ nvim_eval_statusline({str}, {*opts})                  *nvim_eval_statusline()*
                 • use_tabline: (boolean) Evaluate tabline instead of
                   statusline. When true, {winid} is ignored. Mutually
                   exclusive with {use_winbar}.
-                • use_statuscol: (boolean) Evaluate statuscolumn instead of
-                  statusline.
+                • statuscol_lnum: (number) Evaluate statuscolumn for this line
```

Comment:
How about this for more consistency?

```suggestion
                • use_statuscol_lnum: (number) Evaluate statuscolumn for this line
```

Reply:
Fine by me

Reply:
Is it preferred over having both `use_statuscol` and `statuscol_lnum`?

Reply:
> Is it preferred over having both `use_statuscol` and `statuscol_lnum`?
> 
> EDIT: I think it is

Yes, because one implies the other.